zbus/Cargo.toml

```diff
@@ -27,16 +27,16 @@ bus-impl = ["p2p"]
 # Enables API that is only needed for peer-to-peer (p2p) connections.
 p2p = ["dep:rand"]
 async-io = [
-  "dep:async-io",
-  "async-executor",
-  "async-task",
-  "async-lock",
-  "async-fs",
-  # FIXME: We only currently only need this for unix but Cargo doesn't provide a way to enable
-  # features for only specific target OS: https://github.com/rust-lang/cargo/issues/1197.
-  "async-process",
-  "blocking",
-  "futures-util/io",
+    "dep:async-io",
+    "async-executor",
+    "async-task",
+    "async-lock",
+    "async-fs",
+    # FIXME: We only currently only need this for unix but Cargo doesn't provide a way to enable
+    # features for only specific target OS: https://github.com/rust-lang/cargo/issues/1197.
+    "async-process",
+    "blocking",
+    "futures-util/io",
```

Comment:
Please remove all irrelevant changes from the commit/PR. :pray:

Reply:
Whoops, sorry that was unintentional!

Reply:
I am not sure where GitHub is getting this diff from, I don't see it locally!

Reply:
Fixed now.